src/app/views/market/market.component.ts

```diff
@@ -25,6 +25,8 @@ import { TokensFilter } from '@sharedModels/platform-api/requests/tokens/tokens-
 export class MarketComponent implements OnInit {
   iconSizes = IconSizes;
   icons = Icons;
+  tokenOrderByTypes = TokenOrderByTypes;
+  tokenProvisionalTypes = TokenProvisionalTypes;
```

Comment:
same comment, can remove the `=` to just set the type and remove `this.` from below since we're not using these in the template.

Reply:
Done